core/constants.py

```diff
@@ -18,13 +18,23 @@
 
 from __future__ import annotations
 
+import io
 import json
+import os
 import re
-
-from core import python_utils
+import sys
 
 from typing import Any, Dict
 
+_THIRD_PARTY_PATH = os.path.join(os.getcwd(), 'third_party', 'python_libs')
+sys.path.insert(0, _THIRD_PARTY_PATH)
+
+
+class FileNotExist(Exception):
+    """Error class for accessing file which does not exist."""
+
+    pass
```

Comment:
Can we just use `IOError` instead of this?

Reply:
yes Done